homeassistant/components/knx.py

```diff
@@ -120,13 +126,14 @@ def handle_knx_send(event):
             try:
                 knxdata = [int(data) & 0xff]
             except ValueError:
-                _LOGGER.error("KNX data format incorrect")
+                _LOGGER.error("KNX data format incorrect: %s", data)
                 return
 
-        KNXTUNNEL.group_write(knxaddr, knxdata)
+        for addr in knxaddrlist:
+            KNXTUNNEL.group_write(addr, knxdata)
 
     # Listen for when knx_frame_send event is fired
-    hass.bus.listen(EVENT_KNX_FRAME_SEND, handle_knx_send)
+    hass.services.register(DOMAIN, 'group_write', handle_group_write)
```

Comment:
BTW: Is it possible with vol to have an argument that is either a string a a list of strings?